src/Tool/appsettings.json

```diff
@@ -6,15 +6,15 @@
     },
     "AllowedHosts": "*",
     "DefaultPackageVersions": [
-        "Microsoft.Quantum.Standard::0.8.1908.202-beta",
-        "Microsoft.Quantum.Chemistry::0.8.1908.202-beta",
-        "Microsoft.Quantum.Chemistry.Jupyter::0.8.1908.202-beta",
-        "Microsoft.Quantum.Compiler::0.8.1908.202-beta",
-        "Microsoft.Quantum.CsharpGeneration::0.8.1908.202-beta",
-        "Microsoft.Quantum.Numerics::0.8.1908.202-beta",
-        "Microsoft.Quantum.Development.Kit::0.8.1908.202-beta",
-        "Microsoft.Quantum.Research::0.8.1908.202-beta",
-        "Microsoft.Quantum.Simulators::0.8.1908.202-beta",
-        "Microsoft.Quantum.Xunit::0.8.1908.202-beta"
+        "Microsoft.Quantum.Standard::0.9.1908.2906",
```

Comment:
It would be lovely if these version numbers were (possibly environment?) variables all set in one place.